### PR TITLE
Code review test

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
Created the following files:
1.Linter files
2. Html/CSS files
3. .gitignore files
4. MIT Licence file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <a href="#welcome-section"> Contact</a>
+        <a href="#welcome-section"> History</a>
+
+    </div>
+
+  <section id="welcome-section" >
+    <h1> Hey there, this is our collaboaration project</h1>
+    <p id="first-text"> This project would be testing our collaboaration on github and how well we can work together</p>
+    <p> The conetents here based on experimental leraning.</p>
+  </section>
```
Comment (Shedrack-Sunday) [inline index.html:22]: Please kindly use a `nav` bar in place of the `section` tag, to follow best practices on semantics.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <section id="projects" class="project-title">
+    <h1 class="project-header">These are my projects </h1>
+    <a href = "https://github.com/Shedrack-Sunday/Microverse_Trial_Project_Repo_Shedrack_Thomas" ></a>
+
```
Comment (Shedrack-Sunday) [inline index.html:26]: Please makes changes to this section, as the link stated is not same with the repo.

**Review by Shedrack-Sunday — COMMENTED** [review]
Good Job!

- [ ] Nice README 
- [ ] Good Coomit messsages
- [ ]  No linter errors.

However, there are few changes that need to be made, to finish up the projet.

**Review by tugsem — APPROVED** [review]
Hi @Shedrack-Sunday , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights ⭐ 
✅ Correct use of Github feature
✅ Correct use of inline comments
✅ Your comments are polite, clear and reasonable.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
You may need to consider :
- [ ] I highly suggest you to use a couple of emojis and/or gifs in the review.They make the code review more human-friendly 👽🤖

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**